### PR TITLE
Modify conditionals and Hook to load telemetry modal on all pages

### PR DESCRIPTION
**Pull request by bordoni**
[TEC-4851]

Related: https://github.com/the-events-calendar/event-tickets/pull/2711

[TEC-4851]: https://theeventscalendar.atlassian.net/browse/TEC-4851?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by Camwyn — DISMISSED** [review]

File: src/Events/Telemetry/Telemetry.php [hunk 1]
```diff
@@ -195,21 +195,25 @@ public function filter_tec_telemetry_slugs( $slugs ) {
 	 * @return boolean
 	 */
 	public static function is_tec_admin_page(): bool {
+		$currentScreen = get_current_screen();
```
Comment (bordoni) [inline src/Events/Telemetry/Telemetry.php:198]: Naming convention to snakecase

File: src/Events/Telemetry/Telemetry.php [hunk 2]
```diff
@@ -195,21 +195,25 @@ public function filter_tec_telemetry_slugs( $slugs ) {
...
 	public static function is_tec_admin_page(): bool {
+		$current_screen = get_current_screen();
 		$helper = \Tribe__Admin__Helpers::instance();
 
 		// Are we on a tec post-type admin screen?
-		if ( ! $helper->is_post_type_screen( TEC::POSTTYPE ) ) {
+		if (
+			! $helper->is_post_type_screen( TEC::POSTTYPE )
+			//&& ! $helper->is_post_type_screen( TEC::VENUE_POST_TYPE )
+			//&& ! $helper->is_post_type_screen( TEC::ORGANIZER_POST_TYPE )
```
Comment (stratease) [inline src/Events/Telemetry/Telemetry.php:205]:
```suggestion
			! $helper->is_post_type_screen( TEC::POSTTYPE )
```
Commented out section?

**Review by stratease — CHANGES_REQUESTED** [review]
Minor tweak

File: src/Events/Telemetry/Telemetry.php (hunk 2 above)
Comment (Camwyn) [reply src/Events/Telemetry/Telemetry.php:205]: Yeah, I wanted to include those but LG decided it wasn't necessary. I should have actually removed them

**Pull request merged**